vendor/assets/html/cloudinary_cors.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE HTML>
```

Comment:
We prefer not to include the bundled resources in the git repository.
However I will add instructions to fetch them when using the source code.

This folder should be removed before the PR can be merged.

Reply:
I'm happy to remove this. I only did it because your code doesn't work as a `github:` bundled dependency without it. I should have put this on a separate branch so it didn't infect this PR.